tests/common_setup.py

```diff
@@ -99,6 +99,20 @@ def standard_setup_without_client():
 
     set_setup_type(ST_NoClient)
 
+@pytest.fixture(scope="function")
+def setup_with_legacy_client():
+    stop_docker_compose()
+    reset_mender_api()
+
+    docker_compose_cmd("-f ../docker-compose.yml \
+                        -f ../legacy-client.yml \
+                        -f ../docker-compose.storage.minio.yml \
+                        -f ../docker-compose.testing.yml up -d",
```

Comment:
A better way is to keep the `docker-compose.client.yml` and then list `legacy-cient.yml` afterwards. Then it will only override those settings that are specified there.

Reply:
Fixed!